src/pages/manage-my-registrations/update/about-beacon-owner.tsx

```diff
@@ -128,7 +128,9 @@ const EmailAddress: FunctionComponent<FormInputProps> = ({
 
 export const getServerSideProps: GetServerSideProps = withContainer(
   withSession(async (context: BeaconsGetServerSidePropsContext) => {
-    const nextPageUrl = UpdatePageURLs.beaconOwnerAddress;
+    const nextPageUrl =
+      (context.query.review as string) ||
+      UpdatePageURLs.beaconOwnerAddress + context.query.id;
```

Comment:
🤔 what is the `||` for? would `context.query.id` be undefined here since it isn't in the path for this page